ShoppingList/ShoppingList/ShoppingList.csproj

```
@@ -4,6 +4,23 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <UseWPF>true</UseWPF>
+    <Nullable>enable</Nullable>
```

```suggestion
    <Nullable>enable</Nullable>
    <NoWarn>AD0001</NoWarn>
```
I would suggest adding this to your tests project too.

---

Thanks, I forgot about this.